src/gql-validators.ts

```diff
@@ -23,56 +8,3 @@ export interface DocumentError {
 export const failWithError = (err: string) => {
   throw new Error(err);
 };
-
-export const noAnonymousOperations = (
```

Comment:
Being that anonymous operations are being removed, wouldn't it be good to keep this in place? Instead of having it gated behind `loaderOptions.noAnonymousOperations` it could just always run. Unless I'm missing something, it looks like it'd only check the happy path and quietly fail if anonymous operations were added.

Reply:
@zephraph thanks for your notice, I will double check and will try to add unit test for this case.

Reply:
@zephraph just for clarity: you mean that it should fail with error if anonymous operation are used, am I correct?

Reply:
Yep. If they aren’t supported it should fail.